PayPalMessaging/build.gradle

```diff
@@ -7,7 +7,8 @@ android {
     compileSdkVersion rootProject.compileSdkVersion
 
     defaultConfig {
-        minSdkVersion rootProject.minSdkVersion
+        // required to be out of sync with other modules due to messaging SDK min version
+        minSdkVersion 23
```

Comment:
I wonder if instead of hardcoding version 23 here we reference the rootProject such as

`rootProject.midSdkVersionNativeCheckout`

thoughts?

Reply:
I think there could be a future where the min SDK version for PayPal Checkout and the min version for PayPal Messaging dependencies could be different. I'm not necessarily opposed though if we feel that it'd be helpful.

Not sure if this will become a non issue in v5 @sarahkoop?

Reply:
It will become a non-issue in v5 for API 23, but not a bad idea to establish a convention around what to do if one module deviates from the others on API versions. Agreed not to link it with native checkout, but maybe it's own dependency in the top-level gradle file that is `rootProject.minSdkVersionPayPalMessaging`. Then it's more obvious when we do make changes that there are modules that don't align and make sure to update those too.

Reply:
Done: https://github.com/braintree/braintree_android/pull/888/commits/127f32add8131b64fe181921bf65d092afdc29a4